PostHog/Autocapture/PostHogAutoCaptureConfig.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PostHogAutocaptureConfig.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 23/10/2024.
+//
+
+#if os(iOS) || targetEnvironment(macCatalyst)
+    import Foundation
+
+    @objc(PostHogAutocaptureConfig)
+    public class PostHogAutocaptureConfig: NSObject {
```

Comment:
I think they should be disabled by default until its not experimental anymore and we know there are no side effects nor crashing peoples apps

Reply:
nvm this depends on `autocapture` so ignore it.